core/engine/Cargo.toml

```diff
@@ -104,6 +104,7 @@ intrusive-collections = "0.9.6"
 cfg-if = "1.0.0"
 time.workspace = true
 hashbrown.workspace = true
+either = { version = "1", optional = true }
```

Comment:
Can you add the explicit feature? IIRC, implicit features for optional dependencies are [going away](https://github.com/rust-lang/cargo/issues/12826), so it would be good to start defaulting to explicit features.

Reply:
Done.